server/src/test/java/org/elasticsearch/common/hash/MessageDigestsTests.java

```diff
@@ -21,9 +21,11 @@
 
 import org.elasticsearch.test.ESTestCase;
 
+import java.io.File;
```

Comment:
This file seems untouched except for added imports?

Reply:
I am not sure what happened here, I reverted the changes.